source/sass/blocks/advantage.scss

```
@@ -21,5 +25,38 @@
 @media (min-width: $desktop-width) {
   .advantage {
     text-align: center;
+
+    &::before {
+      content: "";
+      height: 357px;
+      width: 281px;
+      background-repeat: no-repeat;
+    }
+
+    &:nth-of-type(1)::before {
+      background-image: url("../../img/iphone-screen-1-desktop@1x.png");
```

```suggestion
      background-image: url("../img/iphone-screen-1-desktop@1x.png");
```
Тут лишний спуск, из-за него эти изображения не отображаются в превью